src/app/static/src/app/store/plottingSelections/getters.ts

```diff
@@ -15,5 +17,12 @@ export const getters = {
             default:
                 return {}
         }
+    },
+
+    downloadFile: () => (data: DownloadPlotData): void => {
```

Comment:
Why is this a getter which returns a method, rather than just making this an action which the component dispatches? Download seems like a classic action!

Also, I'm not convinced 'plottingSelections' is the best module for this, as it doesn't have much to do with plotting selections, since all the data is provided in parameters anyway. Could be new module, or put in root actions?